app/lib/sharezone_plus/page/sharezone_plus_page.dart

```diff
@@ -34,6 +35,14 @@ class SharezonePlusPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final remoteConfig = getRemoteConfiguration();
+    print(
+        'revenuecat_api_key: ${remoteConfig.getString('revenuecat_api_key')}');
+    print(
+        'zeige_pilotschule_karte: ${remoteConfig.getString('zeige_pilotschule_karte')}');
+    print(
+        'useCfHolidayEndpoint: ${remoteConfig.getBool('useCfHolidayEndpoint')}');
```

Comment:
This one is not defined in the defaults and only on the remote config server (locally it should still print false because of the default fallback values of remote config, but lets see)